components/front-page/BgBallFx.tsx

```diff
@@ -0,0 +1,84 @@
+import { motion, useScroll, useSpring, useTransform } from "framer-motion";
+import { useParallax } from "lib/hooks/animation/useParallax";
+import { useRelativeMousePosition } from "lib/hooks/events/useRelativeMousePosition";
+import { useRef } from "react";
+
+export const BgBallGfx = () => {
+  const { scrollYProgress } = useScroll();
+  const ref = useRef<HTMLDivElement>(null);
+  const { x: mx, y: my } = useRelativeMousePosition(ref);
+
+  const osy = useParallax(
+    useSpring(scrollYProgress, {
+      stiffness: 100,
+      damping: 30,
+      restDelta: 0.001,
+    }),
+    220,
+  );
+
+  const osx = useTransform(
+    useSpring(mx, {
+      stiffness: 100,
+      damping: 30,
+      restDelta: 0.001,
+    }),
+    [-1400, 1400],
+    [-50, 50],
+  );
+
+  const iy = useTransform(
+    useSpring(my, {
+      stiffness: 100,
+      damping: 30,
+      restDelta: 0.001,
+    }),
+    [-1400, 1400],
+    [-75, 75],
+  );
+
+  const ix = useTransform(
+    useSpring(mx, {
+      stiffness: 100,
+      damping: 30,
+      restDelta: 0.001,
+    }),
+    [-1400, 1400],
+    [-75, 75],
+  );
+
+  const omy = useTransform(
+    useSpring(my, {
+      stiffness: 100,
+      damping: 30,
+      restDelta: 0.001,
+    }),
+    [-1400, 1400],
+    [-50, 50],
+  );
+
+  const oy = useTransform(() => osy.get() + omy.get());
+
+  return (
```

Comment:
Not sure where this is happening exactly but there is an error in the console happening at `WatchHow.tsx`: `Encountered two children with the same key, ` `. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.`